css/style.css

```diff
@@ -5,6 +5,200 @@ body {
     font-family: 'Roboto', sans-serif;
     -webkit-font-smoothing: antialiased;
 }
+.navigation {
+    background-color: #ffff;
+    position: fixed;
+    width: 100%;
+    border-bottom: 1px solid #DDD;
+    z-index: 9999;
+  }
+  .navigation > .content {
+    height: 6rem;
+    width: 60%;
+  }
+  .content {
+    padding: 0 0.938rem;
+    margin: 0 auto;
+    min-width: 37.5rem;
+  }
+  .navigation .content {
+    display: flex;
+  }
+  .navigation .links {
+    display: flex;
+    color: #838994;
+    font-size: 1.04rem;
+    margin-top: 2.3rem;
+    justify-content: space-around;
+  }
+  ul {
+    list-style: none;
+  }
+  .navigation .links li.active {
+    color: #4c5058;
+    font-weight: bold;
+  }
+  .navigation .links li {
+    display: inline-block;
+    margin-left: 1.563rem;
+  }
+  .navigation .logo {
+    height: 4.375rem;
+    margin-top: 1rem;
+  }
+  .intro {
+    background-image: url('https://raw.githubusercontent.com/CodeYourFuture/karma-clone/master/img/first-background.jpg');
+    background-position: center bottom;
+    background-repeat: no-repeat;
+    background-size: cover;
+    height: 45.938rem;
+    overflow: hidden;
+  }
+  .intro h1 {
+    margin-top: 17.5rem;
+    font-size: 3.125rem;
+    color: white;
+    text-align: center;
+    font-weight: 300;
+  }
+  .intro h3 {
+    margin-top: 1.25rem;
+    font-size: 1.563rem;
+    color: white;
+    text-align: center;
+    font-weight: 300;
+  }
+  .intro .button {
+    color: white;
+    text-align: center;
+    margin-top: 1.563rem;
+  }
+  button {
+    width: 9.688rem;
+    background-color: #f15a29;
+    border-color: #f15a29;
+    text-shadow: none;
+    border-style: none;
+    border-radius: 4px;
+    padding: 1rem 1.5rem;
+    color: white;
+    font-size: 1.125rem;
+  }
+  .cases {
+    background-color: #fff;
+    overflow: hidden;
+    padding-bottom: 5rem;
+  }
+  .cases .content {
+    margin-top: 7.875rem;
+    color: #1d1e21;
+    text-align: center;
+  }
+  
+  .cases h2 {
+    font-size: 2.75rem;
+    margin: 1.625rem 0 3.125rem;
+    font-weight: 300;
+  }
+  .cases .device > li {
+    width: 33.3333333%;
+    float: left;
+    display: flex;
+    justify-content: space-around;
+    flex-direction: column;
+    flex-wrap: wrap-reverse;
+    align-items: center;
+  }
+  .social {
+    background-color: #fff;
+    overflow: auto;
+    padding-bottom: 9.375rem;
+    text-align: center;
+  }
+  .social .icons > li {
+    border-radius: 100%;
+    width: 1.5rem;
+    height: 1.5rem;
+    display: inline-block;
+    margin: 1.25rem 5px;
+    border: 1px solid  #248f8f;
+    padding: 0.625rem 0.625rem 0.625rem 0.625rem;
+    text-align: center;
+    
+  }
+  @media only screen and (max-width: 600px) {
+  
+    body {
+      margin: 0;
+      padding: 0;
+    }
+    .navigation .logo {
+      /* margin-top: -4rem; */
+    }
+    .navigation .links {
+      display: flex
+    }
```

Comment:
on mobile the navigation doesn't fit on the screen. Try to think of a way to fix this (make it smaller? tighter? or disappear completely?)